src/main/python/deep_qa/layers/backend/batch_dot.py

```diff
@@ -123,24 +121,18 @@ def compute_mask(self, inputs, mask=None):
                                      axes=(a_dot_axis, b_dot_axis))
         elif a_dot_axis < b_dot_axis:
             # tensor_a has less dimensions than tensor_b.
-            # We tile tensor_a to have the same shape as tensor_b
+            # We would tile tensor_a to have the same shape as tensor_b,
```

Comment:
hmm i prefer to leave it here, just in case the behavior changes under our feet in the future (and it's thus easier to figure out what went wrong)